packages/format-currency/src/index.js

```diff
@@ -24,11 +24,7 @@ export default function formatCurrency( number, code, options = {} ) {
 	}
 	const { decimal, grouping, precision, symbol } = { ...currencyDefaults, ...options };
 	const sign = number < 0 ? '-' : '';
-	let value = numberFormat( Math.abs( number ), {
-		decimals: precision,
```

Comment:
In the previous version, did we incorrectly pass the last three arguments as an object rather than pass them as separate arguments?

Reply:
No. Previously it called this wrapper function from i18n-calypso, which takes an object
https://github.com/Automattic/wp-calypso/blob/65cee4954be52b54302fe5ffeba47c5549f30929/packages/i18n-calypso/src/i18n.js#L182-L188
That wrapper applies defaults for the decimal point and thousands separator based on the current i18n settings, but every call in format-currency passes values for those options so the i18n-calypso settings don't matter.